Simulation/StandardPlots.m

```diff
@@ -0,0 +1,27 @@
+% This is the standard/default plotting function you can use in your TestCase. 
+% It plots all simulation results to give you a quick overview of your data. 
+% It can also be used as a baseline or template for creating custom plotting functions.
+function StandardPlots(Results)
+  % Hack that helps with graph scaling
+  figure(1, 'position', [0, 0, 1920, 1000]);
+
+  set(groot, 'DefaultLineLineWidth', 5);
+
+  plot(Results.Time, Results.X);
+  hold on;
```

Comment:
Two questions:
1. Do you need to specify hold on every time? Usually in MATLAB you just need one and if you need to turn it off you have to specify `hold off`
2. If the answer is "no" to 1., do you also need to add `hold off`? This usually prevents weird behavior of overlapping plotting on new calls.